docker-compose.yml

```
@@ -0,0 +1,7 @@
+version: "3.8"
+services:
+  web:
+    container_name: memory_game
+    image: inamit323colman/memory_game:latest
```

```suggestion
    image: inamit3231/memory_game:latest
```

We need to change the docker account